sdk/servicebus/azservicebus/exampleSendingMessageBatch_test.go

```diff
@@ -0,0 +1,70 @@
+package azservicebus_test
```

Comment:
Nit: this file should probably be called example_sending_message_batch_test.go

Reply:
Ah, brand new to this, will rename!

Reply:
This does have significance though.  The difference between `azservicebus` and `azservicebus_test` is that the latter will only let you use exported content (i.e. you can't directly access any internal types/funcs).  IMO this is mostly useful when writing examples as it ensures you don't inadvertently use non-public surface area, less so for unit tests.

Reply:
@jhendrixMSFT - agreed, package name was intentional.

I think @seankane-msft is only referring to the filename, however, unless I'm misunderstanding something.

Reply:
I was referring to the filename. I can see why you would want to test from a separate package but its helpful to test internal methods too so I've generally avoided doing so.

Reply:
Derp my bad, yes agreed the file name should be snake-cased, not camel-cased.

Reply:
File renamed, and I'm glad you two were able to work out your differences.